pyomo/opt/tests/base/test_convert.py

```diff
@@ -71,6 +71,7 @@ def write(self,filename="", format=None):
 class OptConvertDebug(unittest.TestCase):
 
     def setUp(self):
+        TempfileManager.push()
```

Comment:
@jsiirola maybe consider push_context() instead of push()?

Reply:
That would definitely be a better name.  Unfortunately, that is a (very)old API.  We could change it, but then would have to add the deprecation paths, etc...  Maybe something to do in a subsequent PR?